compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -433,7 +436,7 @@ object Scanners {
         // in backticks and is a binary operator. Hence, `x` is not classified as a
         // leading infix operator.
         def assumeStartsExpr(lexeme: TokenData) =
-          (canStartExprTokens.contains(lexeme.token) || lexeme.token == COLONeol)
+          (canStartExprTokens.contains(lexeme.token) || lexeme.token == COLONfollow)
```

Comment:
This fix a bug in:
```scala
1
`+` :
  2
```
````+```` is not recognised as a leading operator because followed by `COLONfollow`